src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.view;
+
+import static racingcar.view.constants.Notice.EXCUTOIN_NOTICE;
+import static racingcar.view.constants.Notice.WINNER_NOTICE;
+
+import java.util.List;
+
+public class OutputView {
+
+    public static void printProcessResult() {
+        System.out.println(EXCUTOIN_NOTICE.getMessage());
+    }
+
+    public static void printRoundResult(List<String> carsPositions) {
+        for (String carPosition : carsPositions) {
+            System.out.println(carPosition);
+        }
+        System.out.println();
```

Comment:
```suggestion
    carsPositions.forEach(System.out::println);
        System.out.println();
```
이런식으로 코드를 간결하게 사용할 수 있습니다